backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -2080,15 +2077,14 @@ describe('ContributionResolver', () => {
           // stephen@hawking.uk: [1000, 1000, 1000] - deleted
           // garrick@ollivander.com: [1000, 1000, 1000] - not activated
 
-          describe('user for creation to update does not exist', () => {
+          describe.skip('user for creation to update does not exist', () => {
```

Comment:
I would define that you can update an contribution even tho an user does not exist or is deleted. Therefore I would remove the two tests completely.